README.md

```diff
@@ -40,18 +40,21 @@ To learn more about the cloud-native architecture of ODC go to the [ODC document
 
 _Minimum system requirement per `outsystemscc` instance: 2 GB RAM, 2x 1GHz+ CPU._
 
-To install, use either the binary or Docker option. There are several advantages of running `outsystemscc` as a Docker image as opposed to a binary, if your system supports it:
+To install, use either the binary or Docker option. Run the binary on Linux, or use the Docker image on any OS that supports Docker. Running `outsystemscc` as a Docker image offers several advantages if your system supports it:
 
 * You always run the latest release. You don't need to reinstall each new release.
-* `outsystemscc` can run on Windows or any system that supports Docker.
+* You can run `outsystemscc` on Windows or any system that supports Docker:
+    * You need to install Windows Subsystem for Linux (WSL) on Windows to use the `outsystemscc` Linux binary.
```

Comment:
I still find this line a bit confusing. Is this to show what is the less favorable alternative?
If so, I would add "Otherwise, you would need to (...)"

Reply:
Understood and made the necessary changes.